config/intellij-idea-inspections.xml

```diff
@@ -4944,6 +4951,10 @@
                      enabled_by_default="false"/>
     <inspection_tool class="UnsafeVfsRecursion" enabled="false" level="WARNING"
                      enabled_by_default="false"/>
+    <!-- unstable doesn't mean it isn't safe to use, just might
+         change between versions and isn't stable -->
+    <inspection_tool class="UnstableApiUsage" enabled="false" level="ERROR"
+                     enabled_by_default="false" />
```

Comment:
lets update configuration if it and remove `guava.Beta` from list:
![screen shot 2018-12-02 at 12 27 30 pm](https://user-images.githubusercontent.com/812984/49344552-c8586e80-f62d-11e8-81d2-43a824e38b13.png)

Reply:
Done.